rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java

```diff
@@ -637,29 +637,30 @@ public void acquireAllNodes(Map<String, ?> nodeConfiguration) {
      * @param nodeUrl the url of the node to be released
      */
     public BooleanWrapper removeNode(String nodeUrl, Client initiator) {
-
+        Node node = null;
         //verifying if node is already in the list,
         //node could have fallen between remove request and the confirm
         if (this.nodes.containsKey(nodeUrl)) {
-            logger.info("[" + name + "] removing node : " + nodeUrl);
-            Node node = nodes.remove(nodeUrl);
+            logger.info("Remove node " + nodeUrl + " from node source " + this.name);
```

Comment:
I think it is better to keep the node source logging semantic [nodesourceName] etc

Reply:
Ok no problem, I reverted it to the previous log format. Thanks.